### PR TITLE
GameDB: Final Fantasy X fix

### PR DESCRIPTION
**Pull request by atomic83GitHub**
A long time that I want to post these fix for this major game.

It will need a least 1 test per version (PAL,NTSC,NTSC-J), I have tested the Pal F version and Round mode to negative fix some reverse boss and control issue. The International version is not supported in this PR.

Close https://github.com/PCSX2/pcsx2/issues/760

### CODE REVIEW

**Comment by MrCK1** [comment]
Shawn Murray's Wakeboarding Unleashed hangs at the loading screen, it doesn't go in-gane at all.

**Comment by atomic83GitHub** [comment]
Realy? What is this? https://forums.pcsx2.net/Thread-Core-Bug-Report-Shaun-Murray-s-Pro-Wakeboarder-PAL

**Comment by FlatOutPS2** [comment]
The compatibility surely isn't 5, as what MrCK1 said is true. Even getting past the loading screen the status is more in-game than fully playable.
And though vuClampMode = 2 does reduce some further spikes/flashes, it has not been tested thoroughly, and might as well cause regressions in some parts of the game.

I suggest you don't add gameDB changes for games that you haven't thorougly tested yourself, let alone ones that you haven't played at all. And even when you're sure some fix is necessary, you need to have some backup sources showing the other regions of the game require the same changes. There's no guarantee the NTSC version requires any of these fixes.

**Comment by atomic83GitHub** [comment]
Ok so let's forget about Wakeboarding, I will replace it with a another game that I have and this will need external test: Final Fantasy X : EE round mode to negative for reverse control and boss issue (and some minor compat update).

**Comment by MrCK1** [comment]
By the way, I think we should slow down with the GameDB fixes. Opening more pull requests will only create more merge conflicts for you/us once we approve them ;).

**Comment by atomic83GitHub** [comment]
@MrCK1, this is my last GameDB PR :). I pushed all my fixes and everything is ready for the upcoming 1.6 stable version on my side.

**Comment by FlatOutPS2** [comment]
Are these compatibility ratings you add corroborate by the status on the website? They're not supposed to be changed just because you or some other contributor played the game. Each version/region has its own rating, so you shouldn't change it for all versions when only one is tested either.

I had a quick look at the other recent gameDB changes, and I noticed you changed the name of "Star Wars - Jango Fett" to "Star wars - Bounty Hunter", even though Jango Fett is what the Japanese versions of Bounty Hunter are called. You should probably revert that as that's not something that is done for other games across regions either.

**Comment by lightningterror** [comment]
There won't be any merge conflicts if he doesn't modify the same portion of the code.

Anyway I do agree here, would have been better to have 1 pr with all the changes instead.
As for Star Wars a comment in the DB entity should be enough mentioning they are the same.

**Comment by mirh** [comment]
TBH, the point of all his PR seems "this worked better here - let's extend this universally". 
Not saying it could never actually be the case.. Or that you should complete 100% the game.. But we are on a level that I could just google for "fixes" and call it a day. 

Also wtf flatout are you back?

**Comment by atomic83GitHub** [comment]
@lightningterror , I separated the PR's because some games were not tested at the time. But now I think it's not anymore the case, basicly this PR is ready https://github.com/PCSX2/pcsx2/pull/2137 and this one just need more Fifa test on my side https://github.com/PCSX2/pcsx2/pull/2231 (And by the way, did you finish your test?).

**Comment by FlatOutPS2** [comment]
None of these PRs are ready. You need to remove the compat=5's and check if there's any need to actually add any one these fixes. Some are very likely wrong.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1109,6 +1110,7 @@ Serial = SCED-50642
 Name   = Final Fantasy X [Demo] [Final Fantasy VI PS1 - Bonus Disc]
 Region = PAL-E
 IPUWaitHack = 1
+eeRoundMode = 1 // Fix reverse control and boss in some places
```
Comment (FlatOutPS2) [inline bin/GameIndex.dbf]: There was a discussion about rounding vs clamping for this issue here #760.

**Comment by MrCK1** [comment]
@atomic83GitHub Are you going to update this PR soon to address the changes that @FlatOutPS2 mentioned?

**Comment by FlatOutPS2** [comment]
@atomic83GitHub There's still a compatibility status change for the Korean version of Ratchet & Clank 3 in the commit. And you need to update the commit message to reflect the current status.

**Comment by atomic83GitHub** [comment]
Fixed :).

**Comment by FlatOutPS2** [comment]
One last thing. You should add that it closes this issue: https://github.com/PCSX2/pcsx2/issues/760

**Comment by lightningterror** [comment]
> One last thing. You should add that it closes this issue: #760

That can be done in the PR, and I prefer if it's done in the PR as well, if it's done in the commit then every time you update the commits the issue gets spammed with mentions.

**Comment by refractionpcsx2** [comment]
Done
